packages/splits-sdk/src/constants/index.ts

```diff
@@ -173,6 +175,7 @@ export const ALL_CHAIN_IDS = [
   ...ZORA_CHAIN_IDS,
   ...BASE_CHAIN_IDS,
   ...BLAST_CHAIN_IDS,
+  ...SHAPE_CHAIN_IDS,
 ]
 
 export const SPLITS_SUPPORTED_CHAIN_IDS = [3, 4, 42, ...ALL_CHAIN_IDS]
```

Comment:
is v1 on shape? or just v2? i think these changes will add it to this v1 supported list, right?

Reply:
oh I see, v1 is not on shape, reverting this change

Reply:
does just removing it from there work though? unfortunately i think that might block it from being used in the sdk? i see we pass ALL_CHAIN_IDS into the `supportedChainIds` field in a couple places.

Might need to just hardcode this `SPLITS_SUPPORTED_CHAIN_IDS` list (given it won't change anymore). And then all chain ids does include the new ones?